integration-tests/support/pages/tabs/IntegrationTestsTabPage.ts

```diff
@@ -16,6 +16,10 @@ export class IntegrationTestsTabPage {
     cy.get(integrationTestsTabPO.filterInputField).clear().type(inputString);
   }
 
+  hasIntegrationTest(integrationTestName: string) {
+    cy.get(`[data-id="${integrationTestName}"]`);
```

Comment:
we could move locator to `integrationTestsTabPO`.
Please add `be.visible` and `have.text` assertions.

Reply:
Done!